scripts/test-runner.js

```diff
@@ -314,16 +371,41 @@ function runTest(name, code, options, args) {
     return false;
   }
 }
+
+function testInferiorRepl(replPath) {
```

Comment:
Change this function to "testInOutOfProcessRuntime(runtimePath)" to align with the option name change.

Reply:
Also, the name is a bit confusing to me. Based on its implementation, prepareReplExternalSepc() indicates its purpose better.